docs/developer/index.rst

```diff
@@ -82,6 +80,39 @@ Before you submit a pull request to Bodhi, please ensure that it meets these cri
   `github.com/fedora-infra/bodhi <https://github.com/fedora-infra/bodhi/>`_, or you may e-mail a
   patch to the
   `mailing list <https://lists.fedoraproject.org/archives/list/bodhi@lists.fedorahosted.org/>`_.
+* If your changes contain database migrations, you must add a file named ``summary.migration``
+  in the ``news`` directory, where ``summary`` is a short textual description of the change
+  (or the issue number you're fixing).
+* If it is not present already, add a file in the ``news`` directory named ``username.author``
+  where ``username`` is the first part of your commit's email address, and containing the name
+  you want to be credited as.
+* If you think that the changes you make should be in release notes, add a file in the ``news``
+  directory with the format explained below.
+
+Release Notes
+-------------
+
+To add entries to the release notes, create a file in the ``news`` directory in the
```

Comment:
I would recommend to switch `type` and `source` order, because I always need to check if it's `source.type` or `type.source`. This is caused by type being explained first and then source comes to play.

Reply:
Yeah that's more readable, I used to use the other order because I wanted to introduce the author type first because it was a special case in the source part, but now I've reordered the docs and that's no longer necessary. I'll switch them.